pootle/static/css/style.css

```diff
@@ -1160,6 +1166,15 @@ table.stats td.last-updated
     text-align: right;
 }
 
+html[dir="rtl"] table.stats .number,
+html[dir="rtl"] th.stats-number,
+html[dir="rtl"] td.stats-number,
+html[dir="rtl"] td.stats-percentage,
+html[dir="rtl"] table.stats td.last-updated
```

Comment:
It is aligned exactly the opposite of non-rtl. Perhaps I am misunderstanding you?

![ltr](https://cloud.githubusercontent.com/assets/939880/16868097/0b241676-4a76-11e6-9e9f-4905fc07fc29.png)
![rtl](https://cloud.githubusercontent.com/assets/939880/16868098/0b2ac70a-4a76-11e6-96b1-9e36c47d043b.png)